docs/source/demos/conversion.rst

```diff
@@ -36,7 +36,7 @@ Lets first convert the study level specification and second all the acquisitions
 
   % datalad hirni-spec2bids --anonymize sourcedata/studyspec.json sourcedata/*/studyspec.json
 
-The `anonymize` switch will cause the command to use the anonymized subject identifiers and encode all records of where exactly the data came from into hidden sidecar files, that can tha be excluded from publishing/sharing this dataset.
+The `anonymize` switch will cause the command to use the anonymized subject identifiers and encode all records of where exactly the data came from into hidden sidecar files, that can be excluded from publishing/sharing this dataset.
```

Comment:
This `tha` shouldn't actually be deleted, but replaced by a `then`.

Reply:
ah, gotcha! I changed this and force-pushed